ctpolicy/chromepolicy.go

```diff
@@ -61,6 +62,44 @@ func (chromeP ChromeCTPolicy) LogsByGroup(cert *x509.Certificate, approved *logl
 	return groups, outerror
 }
 
+// LogsByGroup describes submission requirements for embedded SCTs according to
+// https://github.com/chromium/ct-policy/blob/master/ct_policy.md#qualifying-certificate.
+func (chromeP ChromeCTPolicy) LogsByGroup2(cert *x509.Certificate, approved *loglist2.LogList) (LogPolicyData, error) {
+	var outerror error
+	googGroup := LogGroupInfo{Name: "Google-operated", IsBase: false}
+	googGroup.populate2(approved, func(op *loglist2.Operator) bool { return op.GoogleOperated() })
+	if err := googGroup.setMinInclusions(1); err != nil {
+		outerror = err
+	}
+	nonGoogGroup := LogGroupInfo{Name: "Non-Google-operated", IsBase: false}
+	nonGoogGroup.populate2(approved, func(op *loglist2.Operator) bool { return !op.GoogleOperated() })
+	if err := nonGoogGroup.setMinInclusions(1); err != nil {
```

Comment:
Would be nice to have comments throughout to explain why it's ok to proceed anyway despite the errors (to someone coming here in a year, these will look a lot like bugs otherwise).

Reply:
Added the explanation throughout code.